src/assets/styles/App.css

```diff
@@ -0,0 +1,69 @@
+.purple-bg{
```

Comment:
The same App.css file is included twice; please check. Also, please remove the empty CSS files, index.css and style.css.

Reply:
Removed empty style.css and index.css files